.github/workflows/e2e.yaml

```diff
@@ -173,12 +174,11 @@ jobs:
             let circleCIBody = {
               "parameters": circleCIParams,
             };
-            // use pull/<number>/head if it's PR from a fork,
-            // otherwise, always use the current SHA to make sure all CircleCI e2e runnings are using the same commit SHA with this GH Action run
-            if ( "${{ github.event_name }}" == "pull_request" && "${{ github.event.pull_request.head.repo.full_name }}" != "${{ github.repository }}") {
-              circleCIBody["branch"] = '${{ github.ref_name}}'.replace(/^(refs\/)/,"");
+
+            if ( "${{ github.event_name }}" == "pull_request" ) {
+              circleCIBody["branch"] = 'pull/${{ github.event.pull_request.id }}/merge';
```

Comment:
Aren't we testing the branch commit elsewhere?

Reply:
The word `branch` is a way provided by CircleCI to track PR HEAD. Branch commits go to the `else` underneath, which uses `tag` to reference a particular git SHA.